### PR TITLE
streem 201611 (new formula)

### PR DESCRIPTION
**Pull request by yamamuteki**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/streem.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Streem < Formula
+  desc "prototype of stream based programming language"
+  homepage "https://github.com/matz/streem"
+  url "https://github.com/matz/streem/archive/201611.tar.gz"
+  sha256 "807a9ccf3162efb061a15a333943050b329f0232cceb55492e5cd582c09a7fb4"
+  head "https://github.com/matz/streem.git"
+
+  def install
+    system "make"
+    cp_r "bin", prefix
```
Comment (dunn) [inline Formula/streem.rb]: Is there no `make install`? Otherwise, change this to `prefix.install "bin"`.

File: Formula/streem.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Streem < Formula
+  desc "prototype of stream based programming language"
```
Comment (dunn) [inline Formula/streem.rb]: Capitalize "Prototype".

File: Formula/streem.rb (hunk 1 above)
Comment (yamamuteki) [inline Formula/streem.rb]: Thanks. I changed to `prefix.install "bin"` because there was not `make install`.

File: Formula/streem.rb (hunk 2 above)
Comment (yamamuteki) [inline Formula/streem.rb]: Thanks. I fixed capitalize.

File: Formula/streem.rb (hunk 1 above)
Comment (dunn) [inline Formula/streem.rb]: Would you mind asking the developers if they could add an install target?

Comment (yamamuteki) [inline Formula/streem.rb]: I will ask the developers, but can not you just merge with this? Probably I will make it take time, including pull requests.

**Comment by dunn** [comment]
From https://github.com/matz/streem/issues/157:
> Although streem is not stable enough for daily use.

We don't accept formulae until they have stable releases.  Feel free to resubmit when that's available!

**Comment by yamamuteki** [comment]
OK. Thanks.

**Pull request closed without merging**